.pre-commit-config.yaml

```diff
@@ -12,11 +12,11 @@ repos:
         tests/.*.in$
       )$
 - repo: https://github.com/pycqa/isort
-  rev: 5.12.0
+  rev: 5.11.5
   hooks:
   - id: isort
 - repo: https://github.com/ambv/black
-  rev: 23.7.0
+  rev: 23.3.0
   hooks:
     - id: black
```

Comment:
Why not go straight to using `ruff`?

Reply:
I wanted to do the bare minimum to just get tests going for the current state of the code. Since I don't really know WIEN2k and don't have the opportunity to test other than from the outputs I have from the Materials Cloud, this minimizes any silly mistakes. And we can easily go back in the commit logs to figure out what was changed if anything was broken

Reply:
As soon as I make a tag for the AiiDA v1.6 compatible branch, I will start updating everything and switch to `ruff`.